hoomd/md/EvaluatorPairReactionField.h

```diff
@@ -47,15 +47,56 @@ class EvaluatorPairReactionField
     {
     public:
         //! Define the parameter type used by this pair potential evaluator
-        typedef Scalar3 param_type;
+        struct param_type
+            {
+            // potential parameters
+            Scalar eps, eps_rf;
+            bool use_charge;
+
+            #ifdef ENABLE_HIP
+            // set CUDA memory hints
+            void set_memory_hint() const {}
+            #endif
+
+            #ifndef __HIPCC__
+            param_type()
+                {
+                eps = 1;
+                eps_rf = 1;
+                use_charge = false;
+                }
+
+            param_type(pybind11::dict v)
+                {
+                eps = v["epsilon"].cast<Scalar>();
+                eps_rf = v["eps_rf"].cast<Scalar>();
+                use_charge = v["use_charge"].cast<bool>();
+                }
+
+            pybind11::dict asDict()
+                {
+                pybind11::dict v;
+                v["epsilon"] = eps;
+                v["eps_rf"] = eps_rf;
+                v["use_charge"] = use_charge;
+
+                return v;
+                }
+            #endif
+            }
+            #ifdef SINGLE_PRECISION
+            __attribute((aligned(8)));
```

Comment:
Struct size is larger than 8 bytes, so use 16 byte alignment for both single and double precision.